package.json

```diff
@@ -51,7 +51,7 @@
     "minify:js": "uglifyjs public/bundle.js -o public/bundle.js",
     "minify:css": "postcss -u cssnano -i public/bundle.css -o public/bundle.css",
     "watch": "npm-run-all build --parallel watch:*",
-    "watch:js": "watchify ./client.js -o public/bundle.js",
+    "watch:js": "watchify src/assets/js/*.js -o public/bundle.js",
```

Comment:
Unrelated I know but since you're making changes here could you add the [`-v`](https://github.com/substack/watchify#example) flag so watchify logs when it rebuilds the bundle - this is useful during development.